.github/workflows/release-maker.yml

```
@@ -0,0 +1,56 @@
+          else
+            echo "match=false" >> $GITHUB_ENV
+          fi
+
+      - name: Create Tag
+        if: env.match == 'true'
+        id: create_tag
+        run: |
+          tag=$(echo $COMMIT_MESSAGE | grep -oP "$REGEX_PATTERN")
+          echo "tag=$tag" >> $GITHUB_ENV
+
+      - uses: actions/checkout@v2
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)